core/time/shared/gmtime.rb

```diff
@@ -3,8 +3,10 @@
     # Testing with America/Regina here because it doesn't have DST.
     with_timezone("CST", -6) do
       t = Time.local(2007, 1, 9, 6, 0, 0)
+      t.should_not.utc?
       t.send(@method)
       t.should == Time.gm(2007, 1, 9, 12, 0, 0)
+      t.should.utc?
```

Comment:
Yeah, makes sense. Also we may check that `zone` is `UTC` (I mean `t.zone.should == 'UTC'`)

Reply:
Does it make sense not to use `#==` and check instead explicitly datetime parts (minutes, hours, days, etc)?

Reply:
Maybe we should compare using `==` + compare the utc_offset explicitly, possibly adding a helper for that.

Reply:
I followed the suggestion of Andry to check the separate parts of the object. An additional helper would be nice, but I would consider that a future work.